src/main/java/com/google/javascript/gents/Options.java

```diff
@@ -54,6 +54,11 @@
       metaVar = "EXTERNSMAP")
   String externsMapFile = null;
 
+  @Option(name = "--alreadyConvertedPrefix",
+      usage = "Goog.modules starting with this prefix are assumed to be already in TypeScript",
+      metaVar = "ALREADY_CONVERTED_PREFIX")
+  String alreadyConvertedPrefix = "google3";
```

Comment:
A single prefix should work well enough for our usecase. Tsickle converts all .ts modules to goog.modules starting with 'google3', and I don't think any closure native modules start with that. Am I missing something?